lib/widget/input/form_textfield.dart

```diff
@@ -400,6 +413,24 @@ class TextInputState extends FormFieldWidgetState<BaseTextInput>
         ));
   }
 
+  void _updateTextInputController() {
+    widget.controller.maskedValueText = getTextInputValue(isMasked: true);
+    widget.controller.value = getTextInputValue(isMasked: false);
```

Comment:
Just get both unmasked and masked values here once instead of calling a function twice with both true and false?

Reply:
I updated it. Can you check again?